api/explore/next_holiday_calc.js

```diff
@@ -0,0 +1,49 @@
+const Hebcal = require('hebcal');
+const moment = require('moment-holiday');
+
+function nextHoliday(tz) {
+  /* tz = time zone offset recieved from the client side */
+  /* getting the server time time zone offset */
+  const day = new Date();
+  const day20 = new Date();
+  day20.setDate(day.getDate() + 20);
+  if (tz.search('Israel') > 0) {
+    /* assuming Israeli users are jewish.
+        This is obviously false but we'll use it for now */
+    let hday = new Hebcal.HDate();
+    // eslint-disable-next-line id-length
+    for (let i = 0; i < 21; i += 1) {
+      if (hday.holidays()) {
+        return hday.holidays()[0];
+      }
+      hday = hday.next();
+    }
+    return false;
+  }
+  if (tz.search('Argentina') > 0) {
```

Comment:
what are these countries? Why them?

Reply:
I just used all holidays available in moment-holiday